handlers/submit_deposit.go

```diff
@@ -82,3 +101,73 @@ func buildSubmitDepositPageData() (*models.SubmitDepositPageData, time.Duration)
 
 	return pageData, 1 * time.Hour
 }
+
+func handleSubmitDepositPageDataAjax(w http.ResponseWriter, r *http.Request) error {
+	query := r.URL.Query()
+	var pageData interface{}
+
+	switch query.Get("ajax") {
+	case "load_deposits":
+		if r.Method != http.MethodPost {
```

Comment:
Just a minor nit: In theory `/ajax=load_deposits`  should be a `GET` request cause no data is being changed.

Reply:
I'm not sure about this, `GET` data is limited by url size, so it's only suitable for limited amount of information.
In this situation I'm sending in a unbound list of public keys, each consuming 96Bytes in hex format.
When using the GET approach we'll hit url size limits at some point with large deposit files.
eg. 100 pubkeys will already consume 9600 bytes, leaving out any delimiter, argument name or other stuff that'S needed for valid urls ;)

Reply:
Yep, that's a very valid reason to not use GET 👍  Makes sense to move that to the HTTP body.